src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/IFileSystemInfoExtensions.cs

```diff
@@ -11,22 +11,35 @@ namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
     internal static class IFileSystemInfoExtensions
     {
         /// <summary>
-        /// Returns full path to <paramref name="fileSystemInfo"/> including mount point URI.
+        /// Returns full path to <paramref name="fileSystemInfo"/> including mount point URI in a format
+        /// that is suitable for displaying in the CLI or printing to logs.
+        /// This method wraps the path with quotes if it contains spaces.
         /// </summary>
         /// <param name="fileSystemInfo">the file system info to get full path for.</param>
         /// <returns>
         /// Full path to <paramref name="fileSystemInfo"/> including mount point URI.
-        /// If mount point is not a directory, the path is returned as 'mount point URI(path inside mount point)'.</returns>
-        internal static string GetFullPath (this IFileSystemInfo fileSystemInfo)
+        /// If mount point is not a directory, the path is returned as 'mount point URI(path inside mount point)'.
+        /// </returns>
+        internal static string GetDisplayPath(this IFileSystemInfo fileSystemInfo)
         {
+            string result = string.Empty;
             if (fileSystemInfo.MountPoint.EnvironmentSettings.Host.FileSystem.DirectoryExists(fileSystemInfo.MountPoint.MountPointUri))
             {
                 //mount point is a directory, combine paths
-                return Path.Combine(fileSystemInfo.MountPoint.MountPointUri, fileSystemInfo.FullPath.Trim('/', '\\'));
+                result = Path.Combine(fileSystemInfo.MountPoint.MountPointUri, fileSystemInfo.FullPath.Trim('/', '\\'));
+            }
+            else
+            {
+                //assuming file or anything else
+                result = $"{fileSystemInfo.MountPoint.MountPointUri}({fileSystemInfo.FullPath})";
+            }
+
+            if (result.Contains(" "))
```

Comment:
very nit: i'm not sure if replacing spaces are needed for non-folder mount points

Reply:
Is there an example of non-folder mount points with space inside?

Reply:
non folder mount can have spaces inside:
for example: `C:\my nice nuget.nupkg(\my nice template\.template.config\.template.json)`
however in this case we probably should do `"C:\my nice nuget.nupkg"("\my nice template\.template.config\.template.json")`

Reply:
It appears to me that 
- adding quotes specific to each mount point implementation is very tricky and 
- using quotes only at the beginning and end is still desirable to having no quotes.
For these reasons, I will keep this as is. Let me know if you think otherwise.